assets/app/view/game/buy_sell_shares.rb

```diff
@@ -21,6 +21,8 @@ def render
         children = []
 
         if step.current_actions.include?('buy_shares')
+          price_protection = step.protecting_bundle if step.respond_to?(:protecting_bundle)
```

Comment:
should the method be called price_protection as well?

Reply:
Is there a more concise way of writing this in the spirit of `step&.protecting_bundle`?